sql/npc_list.sql

```diff
@@ -456,10 +456,9 @@ INSERT INTO `npc_list` VALUES (16806387,'HomePoint#5','Home Point #5',0,0.000,0.
 INSERT INTO `npc_list` VALUES (16806388,'Ethereal_Junction','Ethereal Junction',0,-357.240,-4.000,240.210,32769,50,50,0,5,116,0,3,0x00008F0900000000000000000000000000000000,0,NULL,0);
 INSERT INTO `npc_list` VALUES (16806389,'Ethereal_Junction','Ethereal Junction',0,-595.160,-3.960,119.090,32769,50,50,0,5,116,0,3,0x00008F0900000000000000000000000000000000,0,NULL,0);
 INSERT INTO `npc_list` VALUES (16806390,'Ethereal_Junction','Ethereal Junction',0,-240.480,-3.970,-75.330,32769,50,50,0,5,116,0,3,0x00008F0900000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (16806391,'qm','???',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (16806392,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (16806393,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (16806394,'Harvesting_Point','Harvesting Point',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
```

Comment:
Did we lose `16806394`?

Reply:
We did.  Does not appear in March update when looking at the mob lists